#!/usr/bin/env python
PACKAGE = "motor_control"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("interface",        str_t,      0,  "CAN Interface",    "can0")
gen.add("id",               int_t,      0,  "ID Number",        0,      0,      64)
gen.add("percent_output",   double_t,   0,  "% Output",         0.0,    -1.0,   1.0)

exit(gen.generate(PACKAGE, "motor_control", "Talon"))